docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,809 @@
+---
+
+In this tutorial you will use a fleet of devices to collect air quality data from different places and display the most recent readings from each device in a custom viewing dashboard.
+By completing this project, you will learn to:
+
+- Configure a fleet of identical machines
+- Organize your fleet using {{< glossary_tooltip term_id="location" text="locations" >}}
+- Collect and sync data from multiple machines
+- Use the Viam TypeScript SDK to query sensor data and create a custom dashboard
+
+  {{<imgproc class="aligncenter" src="/tutorials/air-quality-fleet/three-sensor-dash-wide.png" resize="x900" declaredimensions=true alt="Air quality dashboard in a web browser with PM2.5 readings from three different sensor machines displayed." style="max-width:500px" >}}
+
+## Requirements
+
+You can complete this tutorial using any number of air quality sensing machines.
+
+For each machine, you will need the following hardware:
+
+- [SDS011 Nova PM sensor](https://www.amazon.com/SDS011-Quality-Detection-Conditioning-Monitor/dp/B07FSDMRR5)
+  - If you choose to use a different air quality sensor, you may need to [create your own module](/registry/create/) implementing the [sensor API](/components/sensor/#api) for your specific hardware.
+- A single-board computer (SBC) [capable of running `viam-server`](https://docs.viam.com/get-started/installation/)
+- An appropriate power supply
+
+In addition to `viam-server`, this tutorial uses the following software:
+
+- [`sds0011` sensor module](https://github.com/zaporter/viam-sds011)
+- The [Viam TypeScript SDK](https://ts.viam.dev/)
+
+## Decide how you will organize your fleet
+
+Before you start connecting your devices to the Viam app, you'll need to decide how you want to group your devices.
+
+In the Viam app, {{< glossary_tooltip term_id="machine" text="machines" >}} are grouped into _locations_, and locations are grouped into _organizations_.
+Each location can represent either a physical location or some other conceptual grouping.
+An organization is the highest level grouping, and often contains all the locations (and machines) of an entire company.
+These groupings allow you to manage permissions; you can grant a user access to an individual machine, to all the machines in a location, or to everything in an entire organization.
+You choose how to group your machines.
+
+{{<imgproc class="aligncenter" src="/fleet/fleet.svg" declaredimensions=true alt="Tree diagram with three locations that belong to one organization. One location contains sub-location A containing two machines and sub-location B containing one machine. Each of the two other locations contains machines directly with no sub-locations." >}}
+
+For more information, see [Fleet Management](/fleet/).
+
+### Example
+
+Imagine you create a PM2.5 monitoring company called Pollution Monitoring Made Simple.
```

Comment:
Silly but I had to look up PM2.5, expected?

Reply:
Ah yes, here in Wildfiresville sometimes I forget :D I suppose we don't have to be so specific here! I'm keeping "PM2.5" in some alt text and in the dashboard because that's what the actual readings are, but will make it more conceptual here.
```suggestion
Imagine you create an air quality monitoring company called Pollution Monitoring Made Simple.
```